frontend/src/scenes/data-warehouse/editor/multitabEditorLogic.tsx

```diff
@@ -1028,7 +1028,7 @@ export const multitabEditorLogic = kea<multitabEditorLogicType>([
                 query,
                 variables: Object.fromEntries(
                     Object.entries(values.sourceQuery.source.variables ?? {}).filter(([_, variable]) =>
-                        query.includes(`{variables.${variable.code_name}}`)
+                        query.includes(`variables.${variable.code_name}`)
```

Comment:
I know this was bad, should be using some structured thing here (e.g. wasm hogql parsing), one for later